packages/transition-backend/src/services/importers/PlacesDataSourceImporter.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2025, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import fs from 'fs';
+import { pipeline } from 'stream';
+import { promisify } from 'util';
+import JSONStream from 'JSONStream';
+import { v4 as uuidv4 } from 'uuid';
+import inquirer from 'inquirer';
+
+import placesDbQueries from '../../models/db/places.db.queries';
+import dataSourcesDbQuery from 'chaire-lib-backend/lib/models/db/dataSources.db.queries';
+import { PlaceAttributes } from 'transition-common/lib/services/places/Place';
+import DataSource from 'chaire-lib-common/lib/services/dataSource/DataSource';
+import { fileManager } from 'chaire-lib-backend/lib/utils/filesystem/fileManager';
+import PQueue from 'p-queue';
+
+const pipelineAsync = promisify(pipeline);
+
+// FIXME This prompt works only on command line where the user can answer. Need to support arbitrary prompts, like going through a web interface.
+async function promptOverwriteDataSource(dataSourceName): Promise<boolean> {
```

Comment:
Should this prompt go into the task itself and leave this code path with only parameter that are feeded by various UIs?

Reply:
> Should this prompt go into the task itself and leave this code path with only parameter that are feeded by various UIs?

I suppose it's included in the fixme above. We would need to verify if a datasource already exists first, then request the user somehow if he wants to overwrite it, then call the import. Or else, always ask if we want to overwrite if the datasource exists, without checking its existence first.